packages/client/src/components/SafeSettings/components/Assets/AssetTableView.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { useClipboard } from "hooks";
+import {
+  formatAddress,
+  getFlowscanUrlForContract,
+  parseIdentifier,
+} from "utils";
+import { Copy } from "components/Svg";
+
+const AssetTableView = ({ assets, emptyPlaceholder, onRemoveClick }) => {
+  const clipboard = useClipboard();
+
+  return (
+    <div className="rounded-sm py-3 table-border">
+      {assets.length > 0 ? (
+        assets.map((asset) => {
+          const { contractName, contractAddress } = parseIdentifier(asset);
+          const formattedAddress = formatAddress(contractAddress);
+
+          return (
+            <div className="mx-0 p-3 columns" key={contractName}>
+              <div className="column">{contractName}</div>
+              <div className="column has-text-weight-bold">
+                {formattedAddress}
+                <span
+                  className="pointer"
+                  onClick={() => clipboard.copy(formattedAddress)}
+                >
+                  <Copy className="mt-1 ml-1 pointer" />
+                </span>
+              </div>
+              <a
+                className="column is-flex is-justify-content-end has-text-purple button border-none"
```

Comment:
```suggestion
                className="column button is-transparent"
```

Reply:
I'll update this, tnx! But we still need the is-flex and justify-content-end in order to position this properly